### ISSUE TITLE
Increase verbosity of service module on INFO level

### ISSUE THREAD

**Issue by pasmon** [saltstack/salt#37663]
## Description
I had a state using service module which failed due to "one or more requisite failed". First I checked the minion log but failed to see anything related to that state. Only after I checked the cloud-init-output.log (we are using cloud-init to run highstate the first time) I noticed in the summary that the state failed because some requisite failed. I would like to see something about the failed service state in the minion log as I often look only to that to see what goes on with Salt.

### Setup
Not really the SLS I'm using but should be enough to demonstrate the problem. First state fails for whatever reason (in the example the repo shouldn't exist), and because the first state fails this will happen to the second state too.

	download-example-script:
	  git.latest:
		- name: https://github.com/non-existent/example
		- force_reset : True
		- target: /root/tmp/example

	enable-example-script:
	  file.managed:
		- name: /root/scripts/script.sh
		- source: /root/tmp/example/script.sh

	ensure-service-running:
	  service.running:
		- name: example
		- enable: True
		- full_restart: True
		- watch:
			- file: /root/scripts/*

### Steps to Reproduce Issue
INFO level Salt minion logs do not show anything related to service state "example", only the summary log which is not included in minion log shows the result and some much needed information:

----------
          ID: ensure-service-running
    Function: service.running
        Name: example
      Result: False
     Comment: One or more requisite failed: example.example.enable-example-script
     Changes:
----------

### Versions Report
Salt Version:
           Salt: 2016.3.4

Dependency Versions:
           cffi: 0.8.6
       cherrypy: Not Installed
       dateutil: 1.5
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: 0.21.0
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.8
   mysql-python: Not Installed
      pycparser: 2.14
       pycrypto: 2.6.1
         pygit2: 0.21.4
         Python: 2.7.5 (default, Sep 15 2016, 22:37:39)
   python-gnupg: Not Installed
         PyYAML: 3.10
          PyZMQ: 15.3.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: centos 7.2.1511 Core
        machine: x86_64
        release: 3.10.0-327.36.2.el7.x86_64
         system: Linux
        version: CentOS Linux 7.2.1511 Core

**Comment by Ch3LL** [comment]
So the issue I'm seeing here is not in the service module that we need more verbosity, but with requisities in general, particularly in your example with the watch requisite. You are right even with debug I did not see that state added to the debug log. But when debug was on when I actually had an issue with loading the service module I saw errors. Does this sound like the right approach to this issue to you as well?
